docs/DeveloperGuide.md

```diff
@@ -332,188 +332,306 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
-* 3a. NovaCare detects an invalid Task ID.
+* 1c. At least one of the parameters (name and/ or phone number) are invalid.
 
-    * 3a1. NovaCare requests a valid Task ID.
-    * 3a2. User enters a new Task ID.
+    * 1c1. NovaCare displays an error message specific to the invalid field entry.<br>Use Case ends.
 
-      Steps 3a1-3a2 are repeated until the data entered is correct.
-      Use case resumes at step 2 if patient details are re-entered.
+* 1d. NovaCare detects that the patient already has a registered emergency contact.
+
+    * 1d1. NovaCare displays an error message that the patient already has a registered emergency contact.
+
+      Use case ends.
 
-**Use case: UC05 - Add Emergency contact**
+**Use case: UC07 - Delete Emergency contact**
+**Preconditions: Patient list is not empty**
 
 **MSS**
 
-1. User chooses to add emergency contact details to a selected patient.
-2. NovaCare asks for a patient ID, emergency contact name, and emergency contact number.
-3. User inputs the requested information.
-4. NovaCare adds the emergency contact to the selected patient and displays a confirmation message with the updated emergency contact.
+1. Nurse requests to delete the emergency contact details of a selected patient.
+2. NovaCare deletes the emergency contact details of the selected patient.
 
-    Use case ends.
+   Use case ends.
 
 **Extensions**
 
-* 3a. NovaCare detects that the patient already has a registered emergency contact.
+* 1a. The command format is incorrect.
+
+    * 1a1. NovaCare displays an error message for incorrect command format.<br>Use Case ends.
+
+* 1b. The given index is invalid.
+
+    * 1b1. NovaCare displays an error message specific to the invalid index.<br>Use
+
+* 1d. NovaCare detects that the patient already does not have a registered emergency contact.
 
-    * 3a1. NovaCare notifies the user that the patient already has a registered emergency contact.
+    * 1d1. NovaCare displays an error message that the patient already does not have a registered emergency contact.
 
       Use case ends.
 
-* 3b. NovaCare detects that the patient does not exist.
+**Use Case: UC08 - Add Priority**
+**Preconditions: Patient list is not empty**
 
-    * 3b1. NovaCare notifies the user that the patient does not exist.
-    * 3b2. User either re-enters the correct patient details or cancels the operation.
+**MSS**
+
+1. Nurse requests to add a priority level to a specified patient.
+2. NovaCare updates the given patient details with the entered priority level.
+
+   Use case ends.
+
+**Extensions**
 
-      Use case resumes at step 2 if patient details are re-entered.
+* 1a. The command format is incorrect.
 
-**Use case: UC06 - Delete Emergency contact**
+    * 1a1. NovaCare displays an error message for incorrect command format.<br>Use Case ends.
+
+* 1b. The given index is invalid.
+
+    * 1b1. NovaCare displays an error message specific to the invalid index.<br>Use
```

Comment:
Is there an error here, NovaCare displays an error message specific to the invalid index.<br>Use